README.md

```diff
@@ -18,20 +18,20 @@ For macOS and Linux:
 
 ```bash
 (
- set -x; cd "$(mktemp -d)" &&
- curl -fsSLO "https://storage.googleapis.com/krew-test/krew.zip" &&
- unzip krew.zip &&
- "./build/krew-$(uname | tr '[:upper:]' '[:lower:]')" install krew
+set -x; cd "$(mktemp -d)" &&
```

Comment:
Let’s preserve 2-4 spaces indent

Reply:
Does not work in my shell, I had to remove the spaces.
This makes it compatible. I don't think this will introduce a break in other setups.